bundles/org.openhab.binding.irobot/src/main/java/org/openhab/binding/irobot/internal/dto/MQTTProtocol.java

```diff
@@ -35,23 +34,29 @@
         public int ordered;
         @SerializedName("pmap_id")
         public String pmapId;
+        public String user_pmapv_id;
```

Comment:
Naming convention, seems like it should be something like:
```
@SerializedName("user_pmapv_id")
public String userPmapvId;
```
?

Reply:
@Nuesel : are you still there to finish this PR ?
@jlaur 's comment looks good to me.

Reply:
@lolodomo I already changed the code as proposed by jlaur. But I didn't answer in this chat. Sorry!

Reply:
Apparently not because `user_pmapv_id`; is still in the final proposed code.

Reply:
True, my fault. Hope it is okay, now.